src/main/java/nextstep/courses/domain/Image.java

```diff
@@ -0,0 +1,29 @@
+package nextstep.courses.domain;
+
+public class Image {
+    Integer size;
+    ImageType type;
+    Integer width;
+    Integer height;
+
+    public Image(Integer size, ImageType type, Integer width, Integer height) {
+        validateSize(size);
+        validateRatio(width, height);
+        this.size = size;
+        this.type = type;
+        this.width = width;
+        this.height = height;
+    }
+
+    private void validateSize(Integer size) {
+        if(size > 1024 * 1024){
```

Comment:
`1MB 이하`의 기능적으로 일반화 하여 표현하면 어떨까요?
그것은 어쩌면, `최대 사이즈가 존재한다` 이것이 훨씬 인간의 언어답지 않나요?
적어도 저는 그렇습니다 🤔 

```suggestion
        if(isMaxOverSize(size)){
```

메시지가 발생했군요.
그렇다면, 최대 사이즈 수정시 이제는 어디를 수정하면 좋을지 쉽겠네요 😄 

저는 이런 생각을 하곤합니다.
> `객체지향은 메시지를 보내는것이 어쩌면 가장 중요한 포인트가 아닐까? 메시지가 있기때문에 객체가 발생하는게 아닐까?`